deegree-datastores/deegree-featurestores/deegree-featurestore-sql/src/main/java/org/deegree/feature/persistence/sql/ddl/PostGISDDLCreator.java

```diff
@@ -35,11 +35,7 @@
  ----------------------------------------------------------------------------*/
 package org.deegree.feature.persistence.sql.ddl;
 
-import java.util.ArrayList;
-import java.util.List;
-
-import javax.xml.namespace.QName;
-
+import gcardone.junidecode.Junidecode;
```

Comment:
Seems to be an unused import.

Reply:
Thank you for reporting.
I just removed the unused import.